src/Zip.cc

```diff
@@ -143,15 +143,14 @@ bool Zip::Extract(const std::string &_src,
     // Create intermediate directories if needed.
     std::string dirname = dst;
```

Comment:
Since we're not removing the last character anymore, I think that the `dirname` variable isn't necessary. We could just use `dst` below.